### ISSUE TITLE
Add configuration to use default dune playback insted vod playback.

### ISSUE THREAD

**Issue by newloran2** [newloran2/emplexer#45]
With default dune playback all subtitles bugs and others related with vod playback don't exists.

**Comment by newloran2** [comment]
now setup have a option to use dune default playback or dune vod playback.

**Comment by wgoldman** [comment]
Thank you!  Works perfectly by selecting any movie, any format -- complete subtitle option access.  The only issue I noticed is that if you first press the "Info" button (to see the detailed movie description), you can't play the movie by then pressing the "Play" button.  The error message is "format is not supported."

**Comment by wgoldman** [comment]
Hmm, I may have spoken too soon.  Today I have been unable to play most movie files from within emplexer.  All of these movie files CAN be played directly from the standard Dune interface, but if I try to play using emplexer, the Dune indicates "ERROR" on its LED panel and quits emplexer.  I have tried uninstalling and reinstalling emplexer, including resetting the preferences and paths; I've restarted the Plex Media server and the Dune itself.  My most recent emplexer log is at https://www.dropbox.com/sh/iel34znlqvqk6y8/_X6oNQPkFH
